### PR TITLE
Don't check the links in draft docs

### PR DESCRIPTION
**Pull request by SvenDowideit**
Signed-off-by: Sven Dowideit <SvenDowideit@home.org.au>

### CODE REVIEW

**Comment by moxiegirl** [comment]
Cool. LGTM

File: main.go [hunk 1]
```diff
@@ -73,12 +73,17 @@ func main() {
...
+            fmt.Printf("Draft=%s: SKIPPING %s link check.\n", draft, file)
+        } else {
+            //fmt.Printf("Draft=%s: %s link check.\n", draft, file)
+            err = checkers.CheckMarkdownLinks(reader, file)
+            if err != nil {
+                // this only errors if there is a fatal issue
+                fmt.Printf("ERROR (%s) links: %s\n", file, err)
+                data.AllFiles[file].FormatErrorCount++
+            }
+        }
```
Comment (dnephin) [inline main.go]: I think this is using spaces instead of tabs? The formatting seems to be inconsistent with the rest of the file.

Comment (SvenDowideit) [inline main.go]: it does doesn't it :/ 

guess who was trying MS-Code last night :(

**Comment by SvenDowideit** [comment]
oh fun - changing repo parentage breaks PR's - I'm closing this one, and making a new one

**Pull request closed without merging**